app/src/main/java/vn/asiantech/internship/ui/note/fragment/NewNoteFragment.java

```
@@ -0,0 +1,157 @@
+        switch (v.getId()) {
+            case R.id.imgSaveNote:
+                saveNote();
+                break;
+            case R.id.imgAttachImage:
+                intentGallery();
+                break;
+        }
+    }
+
+    private void intentGallery() {
+        Intent intent = new Intent();
+        intent.setType("image/*");
+        intent.setAction(Intent.ACTION_GET_CONTENT);
+        startActivityForResult(intent, REQUEST_CODE_GALLERY);
+    }
+
+    private boolean copyImageToSDCard(Bitmap image, String path, String fileName) {
+        try {
+            File dir = new File(path);
+            if (!dir.exists()) {
+                dir.mkdirs();
+            }
+
+            File file = new File(path, fileName);
+            if (!file.exists()) {
+                file.createNewFile();
+            }
+            OutputStream fOut = new FileOutputStream(file);
+            image.compress(Bitmap.CompressFormat.PNG, 70, fOut);
+            fOut.flush();
+            fOut.close();
+            return true;
+        } catch (IOException e) {
+            e.getMessage();
+            return false;
+        }
+    }
+
+    private Bitmap getBitmapFromUri(Uri uri) throws IOException {
+        return MediaStore.Images.Media.getBitmap(getActivity().getContentResolver(), uri);
+    }
+
+    private void saveNote() {
+        String title = mEdtTitle.getText().toString();
+        String content = mEdtInputContent.getText().toString();
+        if (!title.isEmpty()) {
+            DatabaseHelper databaseHelper = new DatabaseHelper(getContext());
+            Date now = new Date();
+            SimpleDateFormat dateFormat = new SimpleDateFormat("EEEE", java.util.Locale.getDefault());
+            String dayOfWeek = dateFormat.format(now);
+            String dateTime = new SimpleDateFormat("yyyy/MM/dd").format(Calendar.getInstance().getTime());
+            String time = new SimpleDateFormat("HH:mm:ss").format(Calendar.getInstance().getTime());
```

WARNING: [SimpleDateFormat] To get local formatting use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()`, or use `new SimpleDateFormat(String template, Locale locale)` with for example `Locale.US` for ASCII dates.
 Almost all callers should use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()` to get a ready-made instance of SimpleDateFormat suitable for the user's locale. The main reason you'd create an instance this class directly is because you need to format/parse a specific machine-readable format, in which case you almost certainly want to explicitly ask for US to ensure that you get ASCII digits (rather than, say, Arabic digits).

Therefore, you should either use the form of the SimpleDateFormat constructor where you pass in an explicit locale, such as Locale.US, or use one of the get instance methods, or suppress this error if really know what you are doing.